Plugins/Flow.Launcher.Plugin.BrowserBookmark/Commands/BookmarkLoader.cs

```diff
@@ -1,20 +1,21 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
-using Flow.Launcher.Infrastructure;
 using Flow.Launcher.Plugin.BrowserBookmark.Models;
 using Flow.Launcher.Plugin.SharedModels;
 
 namespace Flow.Launcher.Plugin.BrowserBookmark.Commands
 {
     internal static class BookmarkLoader
     {
+        public static IPublicAPI publicAPI { get; internal set; }
```

Comment:
why we referencing this interface, should it be via context-> PublicAPIInstance?

Reply:
That's because I think this file doesn't need the access for context, so just add the api here.